docs/mgmt/polishing.md

```diff
@@ -183,6 +185,33 @@ public partial class Model
 }
 ```
 
+### Change the format of a property
+
+To assign a new format to a property, you could use this syntax:
+```yaml
+rename-rules:
+  Model.oldProperty: NewProperty|resource-type
+```
+This will rename this property to its new name, and change its format to `resource-type`:
+```diff
+public partial class Model
+{
+    /* other things inside the class */
+
+-    public string OldProperty { get; set; }
++    public ResourceType? NewProperty { get; set; }
+
+    /* other things inside the class */
+}
+```
+
+If only the type of this property needs change, you could omit its new name, like
+```yaml
+rename-rules:
+  Model.oldProperty: -|resource-type
+```
+Please note that the dash and slash `-|` here are mandatory as a placeholder for the property name. The generator uses this symbol to separate the part for property name and its format.
```

Comment:
what's the scenario `name | - `?

Reply:
When you only want to change its name without changing its format which is equivalent to just name without `|-`